### DISCUSSION TITLE
This Serverless Function has crashed. "file not found"

### DISCUSSION THREAD

**Discussion by jalezi** [vercel/community#695] [Help] [answered]
Hi,

I am not very skilled and this is my first time using Vercel. I'm also not very familiar with Next.js.

The problem I have is the following. Everything looks ok locally. There is something wrong with the deployed version. When I try to access dynamic page, I get the error "This Serverless Function has crashed" and in logs "file not found".

The application uses internationalized routing with "next-i18next".

The app is still work in progress, I was just trying to see how Vercel works and now it bothers me that app doesn't work as it does locally.

My good friends google and  stackoverflow couldn't find the answer. I have also looked for the answer on Next.js issues. Maybe I just missed the answer.

I would really appreciate if someone can help me or just point me in the right direction.

[github repo](https://github.com/jalezi/zdravniki-nextjs)

[link to deployed version](https://zdravniki-nextjs.vercel.app/sl/gp/)

```
Operating System:
      Platform: darwin
      Arch: arm64
      Version: Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:29 PDT 2022; root:xnu-8020.121.3~4/RELEASE_ARM64_T8101
 Binaries:
      Node: 16.15.1
      npm: 8.11.0
      Yarn: 1.22.19
      pnpm: N/A
  Relevant packages:
      next: 12.1.5
      react: 18.1.0
      react-dom: 18.1.0
```

vercel log
![Screenshot 2022-07-05 at 15 41 53](https://user-images.githubusercontent.com/44704999/177342655-427fc6c4-66b4-44a4-9c84-c0f42f5b2da3.png)

vercel /gp
![Screenshot 2022-07-05 at 15 19 59](https://user-images.githubusercontent.com/44704999/177342683-8cad448b-8da6-4088-aa5d-878bf9971ae2.png)

vercel /gp/:doctorName
![Screenshot 2022-07-05 at 15 23 19](https://user-images.githubusercontent.com/44704999/177342677-eca4b234-92da-475b-a151-183d400e2ba5.png)

local build
![Screenshot 2022-07-05 at 15 19 16](https://user-images.githubusercontent.com/44704999/177342708-9e696c38-a14d-452a-a346-80954f45bb55.png)

local /gp
![Screenshot 2022-07-05 at 16 24 01](https://user-images.githubusercontent.com/44704999/177351005-23614937-0950-47d7-a29f-91124a533202.png)

local /gp/:doctorName
![Screenshot 2022-07-05 at 16 24 45](https://user-images.githubusercontent.com/44704999/177351175-788357e1-943c-439c-9a05-79fe125217be.png)

**Comment by jalezi** [comment, marked as answer]
Solved.

https://github.com/i18next/next-i18next/issues/1552#issuecomment-981156476

**Reply by amyegan** [reply]
Thank you for coming back to share the answer here to help future people with the same problem! I'm glad you were able to find a solution to this error.

**Reply by jalezi** [reply]
The title and my english is not good enough. It should include: locale, i18, getStaticPaths, etc.
